Makefile

```diff
@@ -1,4 +1,4 @@
-plugins    = ground_truth_slam/ offline_imu_cam/ open_vins/ pose_prediction/ gldemo/ timewarp_gl/ debugview/ audio_pipeline/
+plugins    = ground_truth_slam/ zed/ open_vins/ pose_prediction/ gldemo/ timewarp_gl/ debugview/ audio_pipeline/
```

Comment:
@charmoniumQ we may want to be able to take these plugins as arguments so people can pick and choose what they want a little easier. Editing the make file is a little cumbersome.

Reply:
Marking as resolved, because there is a config-file PR.